scripts/curate_data_axondeepseg_tem.py

```diff
@@ -0,0 +1,162 @@
+import os
+import shutil
+import json
+import argparse
+
+images = {
+    "image.png": "_TEM.png"
+    }
+
+der = {
+    "mask.png": "_TEM_seg-axonmyelin-manual.png",
+    "mask_seg-axon-manual.png": "_TEM_seg-axon-manual.png",
+    "mask_seg-myelin-manual.png": "_TEM_seg-myelin-manual.png"
+    }
+
+
+def get_parameters():
+    parser = argparse.ArgumentParser(description='This script is curating dataset data_axondeepseg_tem to BIDS')
+    parser.add_argument("-d", "--data",
+                        help="Path to folder containing the dataset to be curated",
+                        required=True)
+    parser.add_argument("-o", "--outputdata",
+                        help="Path to output folder",
+                        required=True,
+                        )
+    arguments = parser.parse_args()
+    return arguments
+
+
+def create_json_sidecar(path_folder_sub_id_bids, item_out):
+    data_json = {"PixelSize": [0.00236, 0.00236],
+                 "FieldOfView": [8.88, 5.39],
+                 "BodyPart": "BRAIN",
+                 "BodyPartDetails": "splenium",
+                 "SampleFixation": "2% paraformaldehyde, 2.5% glutaraldehyde",
+                 "Environment": "exvivo"
+                 }
+    item_out = item_out.replace('_TEM.png', '_TEM.json')
+    with open(os.path.join(path_folder_sub_id_bids, item_out), 'w') as json_file:
+        json.dump(data_json, json_file, indent=4)
+
+
+def create_path_sub_id_dir_out(path_sub_id_dir_out):
+    if not os.path.isdir(path_sub_id_dir_out):
+        os.makedirs(path_sub_id_dir_out)
+
+
+def main(root_data, output_data):
+
+    # Remove macOS .DS_Store
+    os.system("find " + root_data + " -name '.DS_Store' -type f -delete")
+
+    # Remove output_data if exists to start clean
+    if os.path.isdir(output_data):
+        shutil.rmtree(output_data)
+    os.makedirs(output_data, exist_ok=True)
+
+    contents_ds = [subdir for subdir in os.listdir(root_data) if os.path.isdir(os.path.join(root_data, subdir))]
+    contents_ds.sort()
+
+    # Loop across contents of each subdirectory
+    for subdir in contents_ds:
+        sub_id = "sub-nyuMouse" + subdir.split('_')[3]
+        sample_id = subdir.split('_')[4]
```

Comment:
You could also write

```suggestion
        _, _, sub_id, sample_id = subdir.split('_')[:4]
        sub_id = "sub-nyuMouse" + sub_id
```

or 

```suggestion
        _, _, sub_id, sample_id, _, _, _, _ = "sub-nyuMouse" + subdir.split('_')
        sub_id = "sub-nyuMouse" + sub_id
```

depending on the number of fields in each filename.